frontend/mr-admin-flate/src/components/forsidekort/Forsidekort.tsx

```diff
@@ -0,0 +1,78 @@
+import styles from "./Forsidekort.module.scss";
+import { Link } from "react-router-dom";
+import { faro } from "@grafana/faro-web-sdk";
+import {
+  FileCheckmarkIcon,
+  HandshakeIcon,
+  RectangleSectionsIcon,
+  TokenIcon,
+} from "@navikt/aksel-icons";
+import { BodyShort, Heading } from "@navikt/ds-react";
+import { erForhandsvisningMiljo } from "../../utils/Utils";
+
+export function Forsidekort() {
+  const forsideKort: {
+    navn: string;
+    ikon: React.ReactNode;
+    url: string;
+    tekst?: string;
+  }[] = [
```

Comment:
Hvis vi først trekker denne komponenten ut i en egen fil så synes det hadde gitt mening at denne dataen var props i stedet for hardkodet inne i komponenten.
Da er også komponenten gjenbrukbar til evt. andre formål også